packages/hdwallet-walletconnect/src/walletconnect.ts

```diff
@@ -122,6 +122,7 @@ export class WalletConnectHDWallet implements core.HDWallet, core.ETHWallet {
   readonly _supportsBSC = false;
   readonly _supportsPolygon = false;
   readonly _supportsGnosis = false;
+  readonly _supportsArbitrum = false;
```

Comment:
Was there a rationale for hdwallet-walletconnect declaring all EVM chains except Ethereum mainnet as unsupported?

Reply:
Consistency sounds sane to me! We can follow-up with an attempt to enable all chains as a follow-up